.storybook/themes/news.scss

```diff
@@ -1636,7 +1636,8 @@
 					"overflow-y": scroll,
 					"overflow-block": scroll,
 					"transform": translate(0, 0),
-					"inline-size": 100%
+					"inline-size": 100%,
+					"height": calc(100vh - var(--header-nav-chain-height))
```

Comment:
We should update the `inline-size` we don't set height with logical properties now.

Reply:
ah! I forgot

Reply:
@nschubach Fixed it in [d6be774](https://github.com/WPMedia/arc-themes-blocks/pull/2132/commits/d6be774721be7681b9e216aa0856f20fceea8ac6)